R/io.R

```diff
@@ -609,6 +610,79 @@ query_civis.character <- function(x, database = NULL, verbose = FALSE, ...) {
   await(queries_get, id = q_id, .verbose = verbose)
 }
 
+#' Export results from a query to S3 and return a file id.
+#'
+#' Exports results from a Redshift SQL query, and returns the id of the file on S3 for use
+#' with \code{\link{read_civis}} or \code{\link{download_civis}}.
+#'
+#' @param x "schema.table", \code{sql("query")}, or a query id.
```

Comment:
By query id, we mean the id from a scripted_sql job, correct?  This might lead to confusion with the queries endpoint.